src/components/InputField/InputField.vue

```diff
@@ -262,10 +285,16 @@ export default {
 
 		&--success {
 			border-color: var(--color-success) !important; //Override hover border color
+			&:focus-visible {
+				box-shadow: rgb(248, 250, 252) 0px 0px 0px 2px, var(--color-primary-element) 0px 0px 0px 4px, rgba(0, 0, 0, 0.05) 0px 1px 2px 0px
+			}
 		}
 
 		&--error {
 			border-color: var(--color-error) !important; //Override hover border color
+			&:focus-visible {
+				box-shadow: rgb(248, 250, 252) 0px 0px 0px 2px, var(--color-primary-element) 0px 0px 0px 4px, rgba(0, 0, 0, 0.05) 0px 1px 2px 0px
+			}
```

Comment:
So I think that we need to standardise how we do focus feedbacks across the board. For the button we chose to use `--color-main-text` because it provides the most contrast with whatever `--colora-main-background` is. Also we considered 2px of background color between the element's border and the focus feedback border but we thought that only "1 border" was sufficient. Whatever we decide, we should decide once and for all and apply it to all or most elements.
cc @jancborchardt @nimishavijay